app/test/check_tx_test.go

```diff
@@ -160,8 +160,9 @@ func TestCheckTx(t *testing.T) {
 			checkType: abci.CheckTxType_New,
 			getTx: func() []byte {
 				signer := createSigner(t, kr, accs[8], encCfg.TxConfig, 9)
-				_, blobs := blobfactory.RandMsgPayForBlobsWithSigner(tmrand.NewRand(), signer.Address().String(), 1_000_000, 1)
-				tx, err := signer.CreatePayForBlob(blobs, opts...)
+				_, blobs := blobfactory.RandMsgPayForBlobsWithSigner(tmrand.NewRand(), signer.Account(accs[8]).Address().String(), 1_000_000, 1)
+				// TODO: go back to the sequences here and handle it
```

Comment:
is this TODO still relevant?

Reply:
I don't think so, I think nina might have added it later